test/hotspot/jtreg/compiler/rangechecks/TestRangeCheckHoistingScaledIV.java

```diff
@@ -28,8 +28,8 @@
  * @library /test/lib /
  * @requires vm.debug & vm.compiler2.enabled & (os.simpleArch == "x64" | os.arch == "aarch64")
  * @modules jdk.incubator.vector
- * @compile --enable-preview -source ${jdk.version} TestRangeCheckHoistingScaledIV.java
- * @run main/othervm --enable-preview compiler.rangechecks.TestRangeCheckHoistingScaledIV
+ * @compile -source ${jdk.version} TestRangeCheckHoistingScaledIV.java
+ * @run main/othervm compiler.rangechecks.TestRangeCheckHoistingScaledIV
```

Comment:
Not important but I assume the @compile line can be removed from a number of tests as it's no longer needed. It was needed for tests that didn't use @enablePreview.

Reply:
Ok, I'll go over all the tests and see if there are `@compile` tags that can be removed